src/hooks/after-create.ts

```diff
@@ -0,0 +1,24 @@
+import { readFileSync, writeFileSync } from 'fs'
+import * as path from 'path'
+import { Hook } from '../hook'
+
+type AfterHookOptions = {
+  projectName: string
+  directoryPath: string
+}
+
+type AfterHookFunction = (options: AfterHookOptions) => void
+
+const afterCreateHook = new Hook<AfterHookFunction>()
+
+afterCreateHook.addHook(
+  'cloudflare-workers',
+  ({ projectName, directoryPath }) => {
+    const wranglerPath = path.join(directoryPath, 'wrangler.toml')
+    const wrangler = readFileSync(wranglerPath, 'utf-8')
+    const rewritten = wrangler.replace('%%PROJECT_NAME%%', projectName)
```

Comment:
`replaceAll` is better. Currently there is only one target `name = "%%PROJECT_NAME%%"`, but more may be added in the future.